merqube_client_lib/templates/equity_baskets/schema.py

```diff
@@ -28,9 +32,10 @@ def get_example(cls) -> dict[str, Any]:
                 adapted from https://github.com/pydantic/pydantic/discussions/4558
         __holiday_spec__"""
         d = {}
-        for field_name, model_field in cls.__fields__.items():
-            if "example" in model_field.field_info.extra:
-                d[field_name] = model_field.field_info.extra["example"]
+        for field_name, model_field in cls.__fields__.items():  # type: ignore
+            logger.info((field_name, model_field))
+            if (jse := getattr(model_field, "json_schema_extra", {})) and (ex := jse.get("example")):
```

Comment:
this field can also be exxplicitly set to None.. hence not just 

ex = getattr(...{}).get(example)